rskj-core/src/main/java/co/rsk/peg/bitcoin/BitcoinUtils.java

```diff
@@ -117,11 +117,27 @@ public static Optional<Sha256Hash> findWitnessCommitment(BtcTransaction tx, Acti
         // If more than one witness commitment, take the last one as the valid one
         List<TransactionOutput> outputsReversed = Lists.reverse(tx.getOutputs());
 
+        if (!activations.isActive(ConsensusRule.RSKIP460)) {
+            return findWitnessCommitmentLegacy(outputsReversed);
+        }
+
+        for (TransactionOutput output : outputsReversed) {
+            byte[] scriptPubKeyBytes = output.getScriptBytes();
+            if (isWitnessCommitment(scriptPubKeyBytes)) {
+                Sha256Hash witnessCommitment = extractWitnessCommitmentHash(scriptPubKeyBytes);
+                return Optional.of(witnessCommitment);
+            }
+        }
+
+        return Optional.empty();
+    }
+
```

Comment:
```suggestion
        for (TransactionOutput output : outputsReversed) {
            byte[] scriptPubKeyBytes = getOutputScriptPubKeyBytes(output, activations);
            if (isWitnessCommitment(scriptPubKeyBytes)) {
                Sha256Hash witnessCommitment = extractWitnessCommitmentHash(scriptPubKeyBytes);
                return Optional.of(witnessCommitment);
            }
        }

        return Optional.empty();
    }

    private static byte[] getOutputScriptPubKeyBytes(TransactionOutput output, ActivationConfig.ForBlock activations) {
        if (!activations.isActive(ConsensusRule.RSKIP460)) {
            return output.getScriptPubKey().getProgram();
        }

        return output.getScriptBytes();
    }

```


wdyt? to avoid having duplicated code

Reply:
Implemented. Thanks